.travis.yml

```diff
@@ -0,0 +1,44 @@
+sudo: required
+
+dist: trusty
+
+language: generic
+
+python:
+  - 2.7
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  matrix:
+    - ROS_DISTRO=indigo
+
+# command to install dependencies
+install:
+  - sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" > /etc/apt/sources.list.d/ros-latest.list'
+  - wget http://packages.ros.org/ros.key -O - | sudo apt-key add -
+  - sudo apt-get update -qq
+  - sudo apt-get install python-rosdep python-catkin-tools -qq
+  - sudo rosdep init
+  - rosdep update
+  # Make a workspace
+  - mkdir -p /tmp/image_pipeline_ws/src
+  # Add this folder to the workspace
+  - ln -s $(pwd) /tmp/image_pipeline_ws/src/image_pipeline
+  # Install dependencies
+  - cd /tmp/image_pipeline_ws/src
+  - git clone https://github.com/ros-perception/vision_opencv.git
+  - (cd vision_opencv && git checkout 7ff3fa72ac05377b9d7c8511ef9378f0e50eff82)
```

Comment:
ok, but the travis file should be agnostic to that. vision_opencv should be installed from deb (and very sorry about the error, I finally got a dev job that was failing (I was waiting for the dev job to be done before releasing)). I am on it.